.github/workflows/main.yml

```diff
@@ -172,6 +173,65 @@ jobs:
       contents: write # to create release
       pull-requests: write # to create release PR
     steps:
-      - uses: google-github-actions/release-please-action@v4
+      - uses: googleapis/release-please-action@v4
         with:
           release-type: node
+
+  deploy-staging:
+    runs-on: ubuntu-latest
+    needs: [ build, check, release-please ]
+    if: ((github.event_name == 'workflow_dispatch' || github.event_name == 'push') && github.ref == 'refs/heads/main' && needs.release-please.outputs.releases_created == 'true')
```

Comment:
won't `release-please.outputs.releases_created` be true when a release-please PR is created, instead of when the release-please PR is merged?

Reply:
According to https://github.com/googleapis/release-please-action?tab=readme-ov-file#outputs, there are two boolean outputs:

- `releases_created`
- `prs_created` 

The release is created once the PR is merged and the release-please action runs.